pathetic-model/src/main/java/org/patheloper/model/snapshot/FailingSnapshotManager.java

```diff
@@ -37,7 +37,7 @@
  */
 public class FailingSnapshotManager implements SnapshotManager {
 
-  private static final Map<UUID, WorldDomain> SNAPSHOTS_MAP = new HashMap<>();
+  private static final ConcurrentMap<UUID, WorldDomain> SNAPSHOTS_MAP = new ConcurrentHashMap<>();
```

Comment:
No need to store this as a ConcurrentMap, we can keep this as a Map here.

Reply:
Another way to allow for multi threaded access is to use a Mutex. But that is more complicated. Instead the concurrent map just takes care of it and doesn’t change the api

Reply:
Done